packages/@react-aria/tag/src/useTagGroup.ts

```diff
@@ -68,7 +68,10 @@ export function useTagGroup<T>(props: AriaTagGroupOptions<T>, state: ListState<T
     direction,
     disabledKeys: state.disabledKeys
   });
-  let {labelProps, fieldProps, descriptionProps, errorMessageProps} = useField(props);
+  let {labelProps, fieldProps, descriptionProps, errorMessageProps} = useField({
+    ...props,
+    labelElementType: 'span'
+  });
```

Comment:
Bug fix. I noticed we were getting a `for` attribute on the label that's rendered as a span, which is invalid html.